pkg/deploy/generator/resources.go

```diff
@@ -8,7 +8,7 @@ import (
 	mgmtkeyvault "github.com/Azure/azure-sdk-for-go/services/keyvault/mgmt/2019-09-01/keyvault"
 	mgmtnetwork "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-08-01/network"
 	mgmtinsights "github.com/Azure/azure-sdk-for-go/services/preview/monitor/mgmt/2018-03-01/insights"
-	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2019-06-01/storage"
+	mgmtstorage "github.com/Azure/azure-sdk-for-go/services/storage/mgmt/2021-09-01/storage"
```

Comment:
L29 needs to be update as well

Reply:
Sorry, I meant the line with Microsoft.Storage

Reply:
Ok, thanks! Then it's done.
I created a new PR to do the whole migration thing as I was scared to break things:
https://github.com/Azure/ARO-RP/pull/3603
Will cleanup this PR later.